docs/react-wiki-archive/API-Extractor.md

```diff
@@ -0,0 +1,17 @@
+### What is API Extractor?
```

Comment:
this should be re-worked/moved to v9 docs

Reply:
pr: https://github.com/microsoft/fluentui/pull/31651